src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -685,6 +685,13 @@ EnsureRelationCanBeDistributed(Oid relationId, Var *distributionColumn,
 	TupleDesc relationDesc = RelationGetDescr(relation);
 	char *relationName = RelationGetRelationName(relation);
 
+	if (relation->rd_rel->relnamespace == PG_CATALOG_NAMESPACE)
```

Comment:
Do we have a test on this case? It might be nice.

Reply:
added